microdnf-plugins/builddep_plugin/builddep.cpp

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include "builddep.hpp"
+
+#include "microdnf/context.hpp"
+#include "utils/string.hpp"
+
+#include "libdnf-cli/output/transaction_table.hpp"
+
+#include <libdnf/base/goal.hpp>
+#include <libdnf/conf/option_string.hpp>
+#include <libdnf/rpm/package_query.hpp>
+#include <rpm/rpmbuild.h>
+
+#include <iostream>
+
+
+namespace fs = std::filesystem;
+
+
+namespace microdnf {
+
+
+using namespace libdnf::cli;
+
+BuildDepCommand::BuildDepCommand(Command & parent) : Command(parent, "builddep") {
+    auto & ctx = static_cast<Context &>(get_session());
+    auto & parser = ctx.get_argument_parser();
+
+    auto & cmd = *get_argument_parser_command();
+    cmd.set_short_description("Install build dependencies for package or spec file");
+
+    auto specs = parser.add_new_positional_arg("specs", ArgumentParser::PositionalArg::UNLIMITED, nullptr, nullptr);
```

Comment:
At least one "specs" argument is required to do something. The same requirement is in DNF4.
So use: `ArgumentParser::PositionalArg::AT_LEAST_ONE`